CRM/Contribute/BAO/Contribution.php

```diff
@@ -3447,6 +3456,12 @@ public static function updateFinancialAccounts(&$params, $context = NULL, $skipT
     }
     if ($context == 'changedAmount' || $context == 'changeFinancialType') {
       $itemAmount = $params['trxnParams']['total_amount'] = $params['trxnParams']['net_amount'] = $params['total_amount'] - $params['prevContribution']->total_amount;
+      if (isset($params['prevContribution']->tax_amount)) {
```

Comment:
@pradpnayak @Edzelopez could you reply to each comment explaining how it is addressed or why you don't think it should be? Thx